Fix some issues of  #3512 and add README.md to librispeech/ssl1 recipe. (#3572)
espnet/espnet: egs2/librispeech/ssl1/README.md

> ## INTRODUCTION

Reviewer: Can you throw a grammar check (e.g., Grammarly)?

This recipe requires fairseq installed, please run:

    cd ${MAIN_ROOT}/tools && make fairseq.done

Run.sh calls hubert.sh, and there are 7 stages in total. First we need to do some data preparation to build espnet-style data/dump folders(stage 0-4). 

    ./run.sh --stage 1 --stop-stage 4
	
Then stage 5 calls script/km.sh to generate psuedo labels used in pretraining. To run this stage, please specify:
    train/valid set, 
    number of k-means clusters, 
    feature type(could be mfcc or hubert extracted feature, will explain later), 
    the percentage you want to used to train the k-means model.
These parameters can be settled in run.sh and pass to hubert.sh

    ./run.sh --stage 5 --stop-stage 5
	
Stage 6 and 7 collect stats of train/valid set and train the hubert model respectively. These two stages have the same functionality as stage 10 and 11 of asr recipes. The only difference is we call `espnet2.bin.hubert_train` instead of `espnet2.bin.train` for training.

    ./run.sh --stage 6 --stop-stage 7
	

> Note that stage 5-7 could run for multiple times, which specified by `pretrain_start_iter` and `pretrain_start_iter`. You may find the default value in run.sh:

Reviewer: ```suggestion
Note that stage 5-7 could run multiple times, which is specified by `pretrain_start_iter` and `pretrain_start_iter`. You may find the default value in run.sh:
```

    pretrain_start_iter=0
    pretrain_stop_iter=2
	

> That refer to 3 iterations, n_clusters_iter[0-2] and feature_iter[0-2] specify the number of clusters and the feature type used for k-means clustering of different iterations. Follow the [Hubert](https://arxiv.org/pdf/2106.07447.pdf) settings of  base model, we use mfcc and 100 cluster for the iteration 0, and extract the latent features from transformer layer 6 of HuBERT model pre-trained in prtrains iteration(HuBERT6), 500 clusters for iteration 1, and (HuBERT9) 500 clusters for iteration 2. Each iterations have different config files. Please refer to conf/tuning/train_asr_hubert_base_960h_pretrain_it*.yaml

Reviewer: ```suggestion
That refers to 3 iterations, n_clusters_iter[0-2] and feature_iter[0-2] specify the number of clusters and the feature type used for k-means clustering of different iterations. Follow the [Hubert](https://arxiv.org/pdf/2106.07447.pdf) settings of the base model, we use mfcc and 100 clusters for the iteration 0, and extract the latent features from transformer layer 6 of HuBERT model pre-trained in prtrains iteration(HuBERT6), 500 clusters for iteration 1, and (HuBERT9) 500 clusters for iteration 2. Each iteration has a different config file. Please refer to conf/tuning/train_asr_hubert_base_960h_pretrain_it*.yaml
```
